src/db/h2/lrsql/h2/sql/ddl.sql

```diff
@@ -156,6 +156,7 @@ CREATE TABLE IF NOT EXISTS lrs_credential (
   api_key    VARCHAR(255) NOT NULL,
   secret_key VARCHAR(255) NOT NULL,
   account_id UUID NOT NULL,
+  UNIQUE (api_key, secret_key),
```

Comment:
was this related to upgrade or just another unassociated improvement?

Reply:
This is upgrade related; when not present, this error occurs:
```
11:38:34.894 [main] INFO  lrsql.system.database - Starting new connection for h2:mem database...
Execution error (JdbcSQLSyntaxErrorException) at org.h2.message.DbException/getJdbcSQLException (DbException.java:632).
Constraint "PRIMARY KEY | UNIQUE (API_KEY, SECRET_KEY)" not found; SQL statement:
CREATE TABLE IF NOT EXISTS credential_to_scope (
  id         UUID NOT NULL PRIMARY KEY,
  api_key    VARCHAR(255) NOT NULL,
  secret_key VARCHAR(255) NOT NULL,
  scope      ENUM('statements/write',
                  'statements/read',
                  'all/read',
                  'all'),   FOREIGN KEY (api_key, secret_key)
    REFERENCES lrs_credential(api_key, secret_key)
    ON DELETE CASCADE
) [90057-202]
```